x/simulation/util.go

```diff
@@ -53,3 +60,66 @@ func mustMarshalJSONIndent(o interface{}) []byte {
 
 	return bz
 }
+
+// OperationInput is a struct that holds all the needed values to generate a tx and deliver it
+type OperationInput struct {
+	R               *rand.Rand
+	App             *baseapp.BaseApp
+	TxGen           client.TxConfig
+	Cdc             *codec.ProtoCodec
+	Msg             sdk.Msg
+	MsgType         string
```

Comment:
I believe MsgType can be removed, since Msg has a .type() method on the interface

Reply:
I tink I can get rid of it.

Reply:
ah i think i cant actually, the interface for sdk.Msg changes, .Type() is no longer there and I can't use the proto url because we haven't transitioned all message types to that.

Reply:
ah didn't realize this happened